### PR TITLE
handle old style update="action" as well as update=(action "action")

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: addon/components/one-way-input.js [hunk 1]
```diff
@@ -77,7 +77,11 @@ const OneWayInputComponent = Component.extend({
     if (this._sanitizedValue !== value) {
       this._sanitizedValue = value;
       if (action) {
-        action(value);
+        if (typeof action === 'string') {
```
Comment (martndemus) [inline addon/components/one-way-input.js]: The invoke action add-on is included, please us that to achieve this.

Comment (kellyselden) [inline addon/components/one-way-input.js]: Changed.

Comment (martndemus) [inline addon/components/one-way-input.js]: That add-on already handles string actions https://github.com/martndemus/ember-invoke-action/blob/master/addon/index.js#L14, could you also implement like it's already done in https://github.com/DockYard/ember-one-way-controls/blob/master/addon/components/one-way-select.js#L104

**Comment by kellyselden** [comment]
@martndemus I think I fixed the action usage now.

**Comment by martndemus** [comment]
@kellyselden Yes, thank you for fixing this!
I'll let @poteto review and merge this.

File: addon/components/one-way-input.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember';
```
Comment (poteto) [inline addon/components/one-way-input.js]: Kill this white line please!

Comment (kellyselden) [inline addon/components/one-way-input.js]: I shall kill it for you! I figured it was desired from here https://github.com/DockYard/ember-one-way-controls/blob/master/addon/components/one-way-select.js#L3

Comment (kellyselden) [inline addon/components/one-way-input.js]: killed

**Comment by poteto** [comment]
Thanks @kellyselden! 

@homu r+

**Pull request merged**